src/doc/rustc/src/platform-support/avr-none.md

```diff
@@ -0,0 +1,78 @@
+# `avr-none`
+
+Series of microcontrollers from Atmel: ATmega8, ATmega328p etc.
+
+**Tier: 3**
+
+## Target maintainers
+
+- [Patryk Wychowaniec](https://github.com/Patryk27) <pwychowaniec@pm.me>
+
+## Requirements
```

Comment:
Could you add a note in this section about a target-cpu being required and maybe link to a list? It's in the sample `rustflags` toml config but having it explicit would be nice.